tests/models/test_bootstrap_election_model.py

```diff
@@ -1063,7 +1063,7 @@ def test_get_national_summary_estimates(bootstrap_election_model, rng):
     bootstrap_election_model.get_aggregate_predictions(
         reporting_units, nonreporting_units, unexpected_units, ["postal_code"], "margin"
     )  # race calling for aggregate prediction interval assumes that the point prediction has been set accordingly
-    lower, upper = bootstrap_election_model.get_aggregate_prediction_intervals(
+    _, _ = bootstrap_election_model.get_aggregate_prediction_intervals(
```

Comment:
I could have sworn I already did this at some point

Reply:
This is the PR of déjà vu 😂 

If it helps, it looks like you made similar changes in PR #109 😄 👍🏻